src/main/scala/ba/sake/hepek/bulma/component/CardComponents.scala

```diff
@@ -51,10 +53,10 @@ case class CardFooter(items: Frag*) extends BulmaElement {
 trait CardComponents {
 
   def card(
-      header: Option[CardHeader],
-      image: Option[CardImage],
+      header: Option[CardHeader] = None,
+      image: Option[CardImage] = None,
       cardContent: CardContent,
```

Comment:
Move this to the front of the parameters list so it can be used without `cardContent =`?

Reply:
Actually I modeled the method signature in this way because in Bulma documentation is shown that the order to build up a Card is: header, image, cardContent and footer.
Beside that, of course, it is possible to modify the order of parameters.